### ISSUE TITLE
What to do on disk fault?

### ISSUE THREAD

**Issue by popovn** [freemansoft/win10-storage-spaces#12]
My config : 1 SSD + 1 HDD

What i must to do if (**when**) my SSD will die?
How to change for another plugged SSD?
All my data will safe ? ( All data on SSD cache does exist on HDD?)

**Comment by rufus20145** [comment]
> All my data will safe ? ( All data on SSD cache does exist on HDD?)

No. When using drive based on Storage Spaces with `Simple` resiliency, data from SSD cache is not mirrored on HDD. And with 1 SSD + 1 HDD you can do either Mirroring (but very limited in size by smallest drive capacity, most likely SSD) or Tiered (with no resiliency at all)

**Comment by rufus20145** [comment]
> How to change for another plugged SSD?

Plug it in, go to Storage Spaces in Control Panel, add it to Pool and rebuild it. Some data will probably be lost.
